src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -748,4 +748,194 @@ public void errorOperationForVector() {
               is(ErrorCode.UNSUPPORTED_UPDATE_FOR_VECTOR.getMessage() + ": " + "$push"));
     }
   }
+
+  @Nested
+  @Order(1)
+  @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+  class VectorSearchExtendedCommands {
+    @Test
+    @Order(1)
+    public void setUp() {
+      insertVectorDocuments();
+    }
+
+    @Test
+    @Order(2)
+    public void findOneAndUpdate() {
+      String json =
+          """
+        {
+          "findOneAndUpdate": {
+            "sort" : {"$vector" : [0.15, 0.1, 0.1, 0.35, 0.55]},
+            "update" : {"$set" : {"status" : "active"}},
```

Comment:
Do we also need to test `findOneAndUpdate()` with `$vector` in projection and `$vector` sort?

Reply:
PR for projection handles projection for $vector. It uses findOneAndUpdate command for the testig.